### DISCUSSION TITLE
Headers localization

### DISCUSSION THREAD

**Discussion by AndreyGrek** [KevinVandy/material-react-table#696] [General]
https://www.material-react-table.com/docs/guides/localization#built-in-locale-examples

In this example, you are replacing one table with another entirely with ready-made localization. But I just need to change the headers, they don’t want to change, as I understand it, due to powerful caching under the hood, are there any solutions? Thank you

**Comment by KevinVandy** [comment]
Try adding the locale as a key prop to force a rerender
